pulsar-common/src/main/java/org/apache/pulsar/common/nar/NarUnpacker.java

```diff
@@ -136,27 +146,27 @@ private static void makeFile(final InputStream inputStream, final File file) thr
     }
 
     /**
-     * Calculates an md5 sum of the specified file.
+     * Calculates an sha256 sum of the specified file.
```

Comment:
What is the reason to use SHA256 instead of md5sum.  The SHA hashing algorithm is much more resource intensive than md5